### PR TITLE
ixia-c example refactoring

### PR DESCRIPTION
**Pull request by Horvath-R**
The original configuration didn't worked.

### CODE REVIEW

File: lab-examples/ixiac01/ixiac01.clab.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ topology:
       kind: keysight_ixia-c-one
       image: ghcr.io/open-traffic-generator/ixia-c-one:0.0.1-2755
       exec:
-        - "./ifcfg add eth1 10.1.1.1 24"
-        - "./ifcfg add eth2 10.2.2.1 24"
+        - "./ifcfg add eth1 1.1.1.1 24"
+        - "./ifcfg add eth2 2.2.2.1 24"
```
Comment (hellt) [inline lab-examples/ixiac01/ixiac01.clab.yml:10]: why chanigng 10 for 1?

**Comment by hellt** [comment]
Hi @bortok can you please review the ixia-c example and update it to the current ixia-c?
It seems it stopped (?) working and go.mod needs to be updated. I'd rather have someone from ixia vetting this integration

File: lab-examples/ixiac01/ixiac01.clab.yml (hunk 1 above)
Comment (Horvath-R) [reply lab-examples/ixiac01/ixiac01.clab.yml:10]: In the file _ipv4_forwarding.go_ test setup is the following, wich is correct, but in the scripts IP addresses are wrong.
Test IPv4 Forwarding with
- Endpoints: OTG **1.1.1.1** -----> 1.1.1.2 DUT **2.2.2.1** ------> OTG 2.2.2.2
- Static Route on DUT: 20.20.20.0/24 -> 2.2.2.2
- TCP flow from OTG: 10.10.10.1 -> 20.20.20.1+

**Comment by bortok** [comment]
hi @hellt looking into this. with the latest `ixia-c-one` we now have ARP working, so the setup can be simplified. how can i push commits into this PR?

**Comment by hellt** [comment]
@bortok  hi can you create a new one!

**Comment by bortok** [comment]
sounds good, will create soon

**Comment by hellt** [comment]
closing this in favor of [keysight-provided](https://github.com/srl-labs/containerlab/pull/1784#issuecomment-1859666807) renewed lab example

**Pull request closed without merging**